src/lxml/objectify.pyx

```diff
@@ -1205,8 +1208,17 @@ cdef dict _PYTYPE_DICT = {}
 cdef dict _SCHEMA_TYPE_DICT = {}
 cdef list _TYPE_CHECKS = []
 
-cdef unicode _lower_bool(b):
-    return u"true" if b else u"false"
+cdef unicode _lower_bool(value):
```

Comment:
Btw., changing arg name here is unrelated, but was required because of cython/cython#4019.